build/build-container.sh

```diff
@@ -61,7 +61,7 @@ if [[ ($# -eq 1 || $# -eq 2 && $2 == "--push" ) ]] && [[ "$1" =~ ^(docs|api|app|
   docker tag nerc/$IMAGE:latest nerc/$IMAGE:$GIT_DESCRIBE
   if [ "$#" -eq 2 ]; then
     echo "Attempting to push image..."
-    docker push nerc/$IMAGE
+    docker push nerc/$IMAGE:$GIT_DESCRIBE
```

Comment:
I don't know whether the docker CLI has changed recently or something! 

Based on the [docker push docs](https://docs.docker.com/engine/reference/commandline/push/) it looks like we could use the `--all-tags` to push both tags at the same time? I.e. I think we could do `docker push nerc/$IMAGE --all-tags`. This is assuming that we get a clean docker registry on each build and don't have every tag for that image we have ever made available when we run the command...